RoddyCore/src/de/dkfz/roddy/core/Analysis.groovy

```diff
@@ -428,59 +431,59 @@ public class Analysis {
      */
     protected void executeRun(ExecutionContext context, boolean preventLoggingOnQueryStatus) {
         withErrorReporting(context, preventLoggingOnQueryStatus, {
-            logger.rare("" + context.getExecutionContextLevel());
-            String datasetID = context.getDataSet().getId();
+            logger.rare("" + context.getExecutionContextLevel())
+            String datasetID = context.getDataSet().getId()
 
-            boolean isExecutable = prepareExecution(context);
+            boolean isExecutable = prepareExecution(context)
 
             if (isExecutable) {
-                boolean successfullyExecuted = false;
+                boolean successfullyExecuted = false
                 try {
-                    boolean execute = true;
+                    boolean execute = true
                     if (context.getExecutionContextLevel().allowedToSubmitJobs) { // Only do these checks, if we are not in query mode!
-                        List<String> invalidPreparedFiles = ExecutionService.getInstance().checkForInaccessiblePreparedFiles(context);
-                        boolean copiedAnalysisToolsAreExecutable = ExecutionService.getInstance().checkCopiedAnalysisTools(context);
-                        boolean ignoreFileChecks = Roddy.isOptionSet(RoddyStartupOptions.disablestrictfilechecks);
-                        execute &= ignoreFileChecks || (invalidPreparedFiles.size() == 0 && copiedAnalysisToolsAreExecutable);
+                        List<String> invalidPreparedFiles = ExecutionService.getInstance().checkForInaccessiblePreparedFiles(context)
+                        boolean copiedAnalysisToolsAreExecutable = ExecutionService.getInstance().checkCopiedAnalysisTools(context)
+                        boolean ignoreFileChecks = Roddy.isOptionSet(RoddyStartupOptions.disablestrictfilechecks)
+                        execute &= ignoreFileChecks || (invalidPreparedFiles.size() == 0 && copiedAnalysisToolsAreExecutable)
                         if (!execute) {
-                            StringBuilder message = new StringBuilder("There were errors after preparing the workflow run for dataset " + datasetID);
+                            StringBuilder message = new StringBuilder("There were errors after preparing the workflow run for dataset " + datasetID)
```

Comment:
See previous comment. In principle just writing "dataset" may be sufficient in a message (compared to `Constants.DATASET`).